src/Core/Framework/Adapter/Command/AssetsPublishCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Command;
+
+use Shopware\Core\Framework\Plugin\Util\AssetService;
+use Shopware\Core\Kernel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AssetsPublishCommand extends Command
+{
+    protected static $defaultName = 'assets:publish';
```

Comment:
Whats the difference to assets:install?

Reply:
`assets:install` is symfony internal and uses `Symfony\Component\Filesystem\Filesystem` instead of flysystem => always local